src/firebase/SCHEMA.md

```diff
@@ -8,33 +8,19 @@ users: {
     creationTime: '',
   }
 },
-postsActive: {
+posts: {
   postId: {
+    sold: bool,
     ownerId: '',
-    ownerName: '',
```

Comment:
I think we should keep ownerName. It will save us reads for each post that we will be displaying.
`
ownerName: auth.currentUser.displayName
`

Reply:
Ah, sorry it was a mistake. Yes we should keep ownerName and delete buyerId.